### PR TITLE
[Doc] Fix Title of the Transformers GLUE example

### PR DESCRIPTION
**Pull request by woshiyyya**
## Why are these changes needed?

Fix the title of the GLUE example, also added missing `ray.data` imports.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [x] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/train/examples/accelerate/accelerate_torch_trainer_no_raydata.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
     set_seed,
 )
 
+import ray.data
```
Comment (krfricke) [inline python/ray/train/examples/accelerate/accelerate_torch_trainer_no_raydata.py:23]: Do we need this here? The file name implies "no ray data"

File: python/ray/train/examples/accelerate/accelerate_torch_trainer.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
     set_seed,
 )
 
+import ray.data
```
Comment (krfricke) [inline python/ray/train/examples/accelerate/accelerate_torch_trainer.py:22]: Does the example not work without this?

**Review by matthewdeng — APPROVED** [review]

File: python/ray/train/examples/accelerate/accelerate_torch_trainer.py (hunk 2 above)
Comment (matthewdeng) [reply python/ray/train/examples/accelerate/accelerate_torch_trainer.py:22]: Ray Data has a lazy import defined [here](https://github.com/ray-project/ray/blob/f3c86d173994e23da21eb3f28c7ddca96fb79c6b/python/ray/__init__.py#L288-L294), but I've never really understood if this should be something done by all libraries, no libraries, or some libraries.

Comment (woshiyyya) [reply python/ray/train/examples/accelerate/accelerate_torch_trainer.py:22]: I've tried that after we `import ray.train`, we can directly call anything methods of `ray.data`.

File: python/ray/train/examples/accelerate/accelerate_torch_trainer_no_raydata.py (hunk 1 above)
Comment (woshiyyya) [reply python/ray/train/examples/accelerate/accelerate_torch_trainer_no_raydata.py:23]: Oh you are right! let me delete this!

File: python/ray/train/examples/accelerate/accelerate_torch_trainer.py (hunk 2 above)
Comment (matthewdeng) [reply python/ray/train/examples/accelerate/accelerate_torch_trainer.py:22]: Ray Data can be accessed after import ray directly. Examples [here](https://docs.ray.io/en/master/data/loading-data.html)

Comment (woshiyyya) [reply python/ray/train/examples/accelerate/accelerate_torch_trainer.py:22]: oh that makes sense now. Ray train import ray, then ray.data is lazily imported.

**Pull request merged**